### ISSUE TITLE
[Feat]: ST-505 ethernet cable connected controller

### ISSUE THREAD

**Issue by 4-26** [mariusz-ostoja-swierczynski/tech-controllers#132]
### The request

ST-505 from Tech Sterowniki works with this integration, but missing few sensors.
Current entities:
![st505-1](https://github.com/user-attachments/assets/16abbb6b-23e4-43ce-8b93-1ff11dd5101d)

Requested entities:
![st505-2](https://github.com/user-attachments/assets/cb06fabb-bc70-4869-b19a-9c6f82448293)

There is strange value on first picrute, first "Temperatura zadana", value floating rarely between -0.6, -0.3 and 0.0.
The second "Temperatura zadana" is always 0.0.

**Comment by anarion80** [comment]
Please provide your actual json data @4-26 as described in https://github.com/mariusz-ostoja-swierczynski/tech-controllers/blob/main/CONTRIBUTING.md#getting-json-data-from-emodul

**Comment by JackTyskie** [comment]
Hi
I have the same problem as [4-26].
app found st-505 controller
but not everything is displayed

[emodul JSON data.txt](https://github.com/user-attachments/files/18482736/emodul.JSON.data.txt)

[home-assistant_tech_2025-01-20T21-17-25.460Z.log](https://github.com/user-attachments/files/18482737/home-assistant_tech_2025-01-20T21-17-25.460Z.log)

**Comment by anarion80** [comment]
The following tile is utilizing both `widgetX` params, while we only ever extract and use `widget`:

```console
{
            "id": 1007,
            "parentId": 0,
            "type": 6,
            "menuId": 2061,
            "orderId": 5,
            "visibility": true,
            "params": {
                "description": "Universal status with widgets",
                "statusId": 1,
                "iconId": 87,
                "widget1": {
                    "txtId": 938,
                    "value": -40,
                    "unit": 7,
                    "type": 1,
                    "params": [
                        0,
                        0,
                        0,
                        0,
                        0
                    ]
                },
                "widget2": {
                    "txtId": 1041,
                    "value": 469,
                    "unit": 7,
                    "type": 9,
                    "params": [
                        0,
                        0,
                        0,
                        0,
                        0
                    ]
                }
            }
        },
```

**Comment by anarion80** [comment]
@4-26  @JackTyskie you can try the version from [tile widgets feature branch](https://github.com/mariusz-ostoja-swierczynski/tech-controllers/tree/feat_tiles_widgets) branch and see if/how it improves things: https://github.com/mariusz-ostoja-swierczynski/tech-controllers/archive/refs/heads/feat_tiles_widgets.zip

**Comment by JackTyskie** [comment]
Hi,

hello
Thank you for your quick response and help.
I will look into it

**Comment by skapron** [comment]
This is working properly.
Now temperatures values are available.

Here are opened discussion.
https://github.com/mariusz-ostoja-swierczynski/tech-controllers/discussions/92

Will be tiles support added to next release?

**Comment by anarion80** [comment]
No, it's not yet working properly: https://github.com/mariusz-ostoja-swierczynski/tech-controllers/issues/144#issuecomment-2613321035. Probably never will without more info from Tech API. But if I can address the comment linked, then I will at least release what will be available then.

**Comment by 4-26** [comment]
![Image](https://github.com/user-attachments/assets/5d92eaa9-5610-492e-8d7e-b3647666f368)
After upgrade: it looks ok.
I think, I will stay on this new version (for some time): 

> [https://github.com/mariusz-ostoja-swierczynski/tech-controllers/archive/refs/heads/feat_tiles_widgets.zip](https://github.com/mariusz-ostoja-swierczynski/tech-controllers/archive/refs/heads/feat_tiles_widgets.zip)

There are still missing few sensors, but they are not so important for me.

Anyway: THANK YOU!
